website/content/docs/external_plugins/_index.md

```diff
@@ -48,6 +48,8 @@ The remaining sections describe all the possible Wash methods that can be passed
 
 **NOTE:** Plugin script invocations run in their own process group (pgrp). Wash will send a SIGTERM signal to the pgrp on a cancelled API/filesystem request. If after five seconds the invocation process has not terminated, then Wash will send a SIGKILL signal.
 
+**NOTE:** Check out the [wash](https://github.com/puppetlabs/wash-ruby) gem. It provides a framework for implementing external plugins in Ruby.
```

Comment:
We have TODO: Link a Ruby example at line 45. We should remove that and maybe move this note there.

Reply:
Done.